src/python/ensembl/production/xrefs/Base.py

```diff
@@ -838,11 +836,11 @@ def get_xref_mapper(self, xref_url: str, species: str, base_path: str, release:
     pep_path = self.get_path(base_path, species, release, 'ensembl', 'peptides.fa');
 
     # Try to find a species-specific mapper first
-    module_name = f'ensembl.xrefs.mapper.{species}'
+    module_name = f'ensembl.xrefs.mappers.{species}'
     class_name = species
     found = importlib.find_loader(module_name)
     if not found:
-      module_name = 'ensembl.xrefs.mapper.BasicMapper'
+      module_name = 'ensembl.xrefs.mappers.BasicMapper'
```

Comment:
Same here. Does it need `production`?

Reply:
mappers module is missing in ensmebl-production repo it exists in a separate repo (https://github.com/Ensembl/ensembl_nf/tree/main/src/python/ensembl/production/xrefs/mappers) 
yes, you're right It is in the production namespace. we need to check with @TamaraNaboulsi on this 
 we will fix this in the release/114 branch